app.rb

```diff
@@ -167,6 +168,16 @@
   erb :term
 end
 
+get '/reports/:country' do
+  redirect to('/login') unless current_user
+  @country = Everypolitician.country(slug: params[:country])
+  stats_raw = JSON.parse(open('https://everypolitician.github.io/gender-balance-country-stats/stats.json').read, symbolize_names: true)
```

Comment:
It would be nice to do something a little more defensive in case this fails for some reason.